### DISCUSSION TITLE
hlep! I can't open Demo in version 3.9.1.

### DISCUSSION THREAD

**Discussion by Sakanamii** [akiver/cs-demo-manager#947] [Q&A] [answered]
![bug](https://github.com/user-attachments/assets/c1a9d377-a405-4bd1-8ea8-ff5ca1de95e8)

**Comment by Sakanamii** [comment, marked as answer]
Solved

**Reply by akiver** [reply]
Please search for similar issues, you would have found https://github.com/akiver/cs-demo-manager/issues/943
